test/e2e/basic_volume_io/basic_volume_io_test.go

```diff
@@ -28,8 +29,12 @@ func TestBasicVolumeIO(t *testing.T) {
 	RunSpecsWithDefaultAndCustomReporters(t, "Basic volume IO tests, NVMe-oF TCP and iSCSI", rep.GetReporters("basic-volume-io"))
 }
 
-func basicVolumeIOTest(scName string) {
-	volName := "basic-vol-io-test-" + scName
+func basicVolumeIOTest(protocol common.ShareProto) {
+	scName := "basic-vol-io-test-" + string(protocol)
+	err := common.MkStorageClass(scName, e2e_config.GetConfig().BasicVolumeIO.Replicas, protocol)
```

Comment:
Does this delete a storage class if it already exists? Eg if a test failed to cleanup?

Reply:
No, and I don't think it should cause we risk ignoring other issues.
If a test fails to cleanup on a successful run then we should notice it testing before the commit, during the CI run, or other runs.
We shouldn't cleanup on failure as that can hamper post mortem analysis on the live system.